.github/ui-e2e-tests/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "@traceable/e2e",
+  "name": "@hypertrace/e2e",
   "version": "0.0.0",
-  "description": "E2E Tests for Traceable UI App",
+  "description": "E2E Tests for hypertrace UI App",
   "author": "",
   "license": "MIT",
```

Comment:
Shouldn't the LICENSE be https://github.com/hypertrace/hypertrace-ui/blob/main/package.json#L8?

Reply:
https://github.com/npm/npm/issues/8795

Reply:
Ohh. will take for that in separate PR.